### DISCUSSION TITLE
Beginner Question. Project Name and CSS generation

### DISCUSSION THREAD

**Discussion by ashishsinha24** [microsoft/fluentui-blazor#1287] [General]
Hi,

I am trying to incorporate FluentUI in my exisiting project. 
After going through the documents, and failing miserably after trying several variations

<link rel="stylesheet" href="IdfBlaze.Server.styles.css" />
<link rel="stylesheet" href="IdfBlaze.IdfBlaze.Server.styles.css" />

What is the proper way to use the css include statement?

![image](https://github.com/microsoft/fluentui-blazor/assets/8043838/7fbcbcc7-f4ac-4b80-b225-37e43aeb0b33)

**Comment by vnbaaij** [comment]
Sorry, not quite clear what you are asking. Which CSS are you trying to include?

**Reply by ashishsinha24** [reply]
As documented in https://learn.microsoft.com/en-us/fluent-ui/web-components/integrations/blazor

`<link href="{PROJECT_NAME}.styles.css" rel="stylesheet" /> `

I am unable to get this working. Creating a new project works fine. 
But, I am trying to switch over from a 3rd party library to FluentUI and I have to use it in an existing project.

**Reply by vnbaaij** [reply]
Don't use that documentation. It is way behind the current guidelines. Please use instructions as provided on https://www.fluentui-blazor.net/

That CSS is just a standard Blazor thing. Not Fluent specific. See https://learn.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-8.0

**Reply by ashishsinha24** [reply]
Thanks . Got it working
